config/http/handler/handlers/storage-description.json

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^5.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": "The suffix appended to a storage container to find its description resource.",
+      "@id": "urn:solid-server:default:variable:storageDescriptionSuffix",
+      "valueRaw": ".well-known/solid"
```

Comment:
There seems to be a mixture of leading and no leading slash in this file.

I also don't understand what the suffix is. `.well-known/solid` is a prefix? Or is it the relative path that we mean?

Reply:
Yes it's a relative path, relative to a storage root (so not the base URL).

Reply:
I would name it like that then. It's not really a suffix.